test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
After further investigation, turns out I was wrong, it's not the same user object. It's just that all the users have the same `_id`. (╯°□°）╯︵ ┻━┻)